secure_redis/migrations/0001_initial.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
```

Comment:
Actually I was wondering which one to use `migration` vs `command`. I that you confused since you didn't see any model in there but the reasons I decided to make it as migration:
1. Its easier
2. Executed one time only
3. Its doing a data migration from old to new form
@danaspiegel What do you think?